fsm.go

```diff
@@ -102,6 +102,9 @@ func (s *stateMachine) tick() ([]packet, []Update) {
 		ps = append(ps, s.makeMessagePing(m))
 	}
 
+	if len(s.ml.members) == 0 {
```

Comment:
Consider checking `s.pingTarget == ""` here instead, more directly what it needs to know.